submissions/emmanuelEsuong/htmlForm/signupPage.html

```diff
@@ -1,88 +1,130 @@
 <!DOCTYPE html>
-
 <html lang="en">
-
+    <!--header-->
 <head>
     <meta charset="UTF-8">
     <title>SignUp</title>
+        <!--links-->
+        <link rel="stylesheet" type="text/css" href="../css/styles.css">
+        <link rel="stylesheet" type="text/css" media="screen" href="../css/styles-mediaQuery.css">
+        <link rel="stylesheet" href="https://cdn.rawgit.com/vaakash/socializer/80391a50/css/socializer.min.css">
```

Comment:
What does this cdn `https://cdn.rawgit.com/vaakash/socializer/80391a50/css/socializer.min.css` do for you

Reply:
https://cdn.rawgit.com/vaakash/socializer/80391a50/css/socializer.min.css

It adds icons to the page (i.e phone and envelope icons) but it's for the "contact - us" page, pasted it here by mistake.

Reply:
I think the font awesome CDN which you are using below, already have social media icons

Reply:
Both links are associated with the icons.